packages/HorizonUI/Sources/HorizonUI/Sources/Components/ProgressBar/HorizonUI.ProgressBar.swift

```diff
@@ -106,17 +106,20 @@ extension HorizonUI.ProgressBar {
     }
 
     private var backgroundView: some View {
-        Capsule()
+        Rectangle()
             .fill(size.backgroundColor)
-            .overlay {
-                Capsule()
-                    .stroke(size == .small ? .clear : progressColor, lineWidth: 2)
-            }
+            .huiCornerRadius(level: .level28)
+            .huiBorder(
+                level: .level2,
+                color: size == .small ? .clear : progressColor,
+                radius: size.height
```

Comment:
Please use the same radius as the `level6` corner radius. Take a look at `Pill` to see how it could be done. You could save a private property of corner radius, then reference its value.